MinVer.Lib/MinVer.Lib.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <NoWarn>$(NoWarn);CA1716</NoWarn>

```

Comment:
@damianh says this is OK. :trollface:

Reply:
Approved.

Now I want to see style cop analysis.